batchflow/models/tf/base.py

```diff
@@ -1472,18 +1439,19 @@ def _dynamic_crop(cls, inputs, static_shape, dynamic_shape, data_format='channel
 
     @classmethod
     def initial_block(cls, inputs, name='initial_block', **kwargs):
-        """ Transform inputs with a convolution block
+        """ Transform inputs with a convolution block. Usually used for initial preprocessing,
```

Comment:
IMO convolution is a detail of realisation here
```suggestion
        """ Transform inputs. Usually used for initial preprocessing,
```